docs/send-data/collect-from-other-data-sources/azure-blob-storage/troubleshoot-azure-blob-storage-log-collection.md

```diff
@@ -110,58 +110,63 @@ For more information about using Live Tail, see [Live Tail](/docs/search/live-ta
 
 ### Common Azure function errors
 
-`ExitCode C0000005
+ExitCode C0000005
 
 ExitCodeString NATIVE ACCESS VIOLATION
 
-Managed Exception = System.AccessViolationException:Attempted to read or
-write protected memory. This is often an indication that other memory is corrupt.
+Managed Exception = System.AccessViolationException:Attempted to read or write protected memory. This is often an indication that other memory is corrupt.
 
-CallStack - Managed Exception`
+CallStack - Managed Exception
```

Comment:
Why were the tick marks removed from this error message formatting?

Reply:
It was not working with the ticks present. Now I have modified it accordingly by adding three ticks.